app/graphql/types/submission_type.rb

```diff
@@ -31,5 +31,14 @@ class SubmissionType < Types::BaseObject
     field :user_id, String, null: false
     field :width, String, null: true
     field :year, String, null: true
+
+    field :offers, [Types::OfferType], null: false do
+      argument :gravity_partner_id, ID, required: true
```

Comment:
Yeah, we often do something like this in Reaction where we request different image sizes, so something like 

```
image(size: "large")
```